AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Smoke_GPU.py

```diff
@@ -0,0 +1,46 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+import os
+import pytest
+
+from ly_test_tools.o3de.editor_test import EditorSingleTest, EditorTestSuite, EditorBatchedTest
+
+# File with the list of vulkan validation errors to ignore
+# can be found in 'tests\rhi_validation\vulkan_skip_errors.py'
+
+@pytest.mark.SUITE_smoke
+@pytest.mark.REQUIRES_gpu
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+class TestAutomation_RHIValidation_Vulkan(EditorTestSuite):
+    # Remove -BatchMode from global_extra_cmdline_args since we need rendering for these tests.
+    global_extra_cmdline_args = [
+        "-autotest_mode",
+        "-rhi=vulkan",
+        "-rhi-device-validation=enable"
+        ]  # Default is ["-BatchMode", "-autotest_mode"]
+    use_null_renderer = False  # Default is True
+
+    class test_RHIValidation_Vulkan_DefaultLevel(EditorBatchedTest):
+        from .tests.rhi_validation import RHIValidation_Vulkan_DefaultLevel as test_module
```

Comment:
I'm not sure this is needed, we already have some tests that open the editor for vulkan.
AutomatedTesting/Gem/PythonTests/Performance/TestSuite_Periodic_Vulkan.py.

The low end pipeline is not already being tested so that is needed.

Reply:
I've checked that file `AutomatedTesting/Gem/PythonTests/Performance/TestSuite_Periodic_Vulkan.py` it contains tests focusing on performance and since it's periodic it's not meant to run on AR, but its focus is on nightly build. Those tests also do not enable the validation layer, so it's not checking for vulkan validation errors.

The goal for this PR is to add smoke tests (for now in nightly build, but eventually in AR) that are fast, enable the thi validation layer and check if validation errors appear in the log, which is a different goal than the periodic performance tests.